lxd/storage/utils.go

```diff
@@ -225,6 +227,7 @@ func LXDUsesPool(dbObj *db.Cluster, onDiskPoolName string, driver string, onDisk
 }
 
 // MakeFSType creates the provided filesystem.
+// Deprecated: Use drivers.MakeFSType.
```

Comment:
@monstermunchkin is it worth just updating the use of this function to use the new one in the storage package?

Reply:
We could do that. I just wanted to focus on the new driver for now.

Reply:
@monstermunchkin it would help to avoid duplication and we may forget to remove it when we remove the old storage layer in the future. It looks like its only used in 6 places so shouldn't be a big change to switch over.